tools/tpm2_createprimary.c

```diff
@@ -216,6 +217,7 @@ execute_tool (int               argc,
         g_flag = 0,
         G_flag = 0,
         C_flag = 0;
+    bool E_flag = false;
```

Comment:
The name E_flag is ok here, but when passed along as a function argument, it should have a more meaningful name.